gql_code_builder/lib/src/operation/data.dart

```diff
@@ -183,7 +261,7 @@ Method _buildGetter(
     return Method(
       (b) => b
         ..returns = returns
-        ..name = name
+        ..name = name.startsWith("_") ? "\$$name" : name
```

Comment:
`common.dart` contains a `identifier` util. Could you add this logic to that util and use it here?

Reply:
done in https://github.com/gql-dart/gql/pull/32/commits/1438046072a7a213458324125bbd8e33502d857b

Reply:
👍 

One day, I'll have to apply this everywhere.